### DISCUSSION TITLE
my vercel account is blocked every creation

### DISCUSSION THREAD

**Discussion by viral0790** [vercel/community#4006] [Account Issue]
### Question

my vercel account is blocked every creation

**Comment by amyegan** [comment]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
